### DISCUSSION TITLE
turn / stun servers

### DISCUSSION THREAD

**Discussion by darrius** [subins2000/p2pt#41] [Q&A]
Hi, sorry for the newbie question, but do you need to use turn servers with p2pt? This would help when peers are behind firewalls and can't connect directly correct?

**Comment by subins2000** [comment]
There is no specific requirement for TURN for p2pt to work. You are free to add a TURN server by using `config` in simple-peer : https://github.com/feross/simple-peer#connection-does-not-work-on-some-networks

Then make a custom class wrapper over P2PT like this https://github.com/subins2000/p2pt/blob/master/node.js In this, `wrtc` config is overridden.

**Comment by darrius** [comment]
Thank you the clarification. Makes sense.
